readme.md

```diff
@@ -123,44 +154,83 @@ const url = 'https://upload.wikimedia.org/wikipedia/en/a/a9/Example.jpg';
 
 ## API
 
-### fileType(input)
 
-Returns an `object` with:
+### FileType.fromBuffer(buffer)
+Detect the file type of a `Buffer`/`Uint8Array`/`ArrayBuffer`.
+The file type is detected by checking the [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)#Magic_numbers_in_files) of the buffer.
+
+If file access is available, it is recommended to use `fromFile()` instead
+
+| Parameter | Type                                                                                                                                                                                                                                                                            | Description                       |
+|-----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------|
+| path      | [`Buffer`](https://nodejs.org/api/buffer.html) or [`Uint8Array`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint8Array)  or [`ArrayBuffer`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer) | Buffer representing file content  |
```

Comment:
I don't want to use Markdown tables. They're annoying to maintain.

Reply:
Converted to nested list